aim/agent/aid/universes/aci/tenant.py

```diff
@@ -302,18 +302,19 @@ def reset_ownership(self):
 
 class AciTenantManager(utils.AIMThread):
 
-    def __init__(self, tenant_name, apic_config, apic_session, ws_context,
+    def __init__(self, tenant_name, apic_config, ws_context,
                  creation_succeeded=None, creation_failed=None,
                  aim_system_id=None, get_resources=None, *args, **kwargs):
         super(AciTenantManager, self).__init__(*args, **kwargs)
         LOG.info("Init manager for tenant %s" % tenant_name)
         self.get_resources = get_resources
         self.apic_config = apic_config
         # Each tenant has its own sessions
-        self.aci_session = apic_session
         self.dn_manager = apic_client.DNManager()
         self.tenant_name = tenant_name
-        children_mos = get_children_mos(self.aci_session, self.tenant_name)
+        ws_context.establish_aci_session()
```

Comment:
Is this needed? Isn't the session established already in the WebSocketContext class?